### ISSUE TITLE
Homebridge securitysystem

### ISSUE THREAD

**Issue by nfstange** [hoobs-org/HOOBS#550]
**Description**
"Error: The requested accessory 'Security system' was not registered by any plugin."

**Expected behavoir**
Will not allow HOOBS to start backup 

**Plugin name**
homebridge-securitysystem

**Please report the issue to the plugin developer**
We like to help the community, however some issues are out of our control. Please report this to the plugin developer and include a link here.

**Version**
3.1.23

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [X ] No

**Pervious version**
If you upgraded, please let us know your previous version.

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ X] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [X ] HOOBS SD Card
- [ ] Custom Install

**If this is a custom install, what's your operating system?**
Please let us know what operating system and version you are using. Ex. Fedora 30 or macOS Catalina.

**Post your config**
{
            "accessory": "Security system",
            "plugin_map": {
                "plugin_name": "homebridge-securitysystem",
                "index": 0
            },
            "name": "Security system",
            "default_mode": "Off",
            "arm_seconds": 0,
            "trigger_seconds": 0,
            "siren_switch": true,
            "override_off": false,
            "save_state": false,
            "server_arm_delay": false
        }

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
[Homebridge] [4/15/2020, 16:52:12] Loaded config.json with 7 accessories and 3 platforms.
[Homebridge] [4/15/2020, 16:52:13] Warning: skipping plugin found at '/home/hoobs/.hoobs/node_modules/@piets/homebridge-samsung-tizen' since we already loaded the same plugin from '/home/hoobs/.hoobs/node_modules/homebridge-samsung-tizen'.
[Homebridge] [4/15/2020, 16:52:14] Loaded plugin: "homebridge-camera-ffmpeg"
[Homebridge] [4/15/2020, 16:52:14] Registering platform "homebridge-camera-ffmpeg.Camera-ffmpeg"
[Homebridge] [4/15/2020, 16:52:14] Loaded plugin: "homebridge-roborock"
[Homebridge] [4/15/2020, 16:52:14] Registering accessory "homebridge-roborock.RoborockVacuum"
[Homebridge] [4/15/2020, 16:52:14] Loaded plugin: "homebridge-samsung-tizen"
[Homebridge] [4/15/2020, 16:52:14] Registering platform "homebridge-samsung-tizen.SamsungTizen"
[Homebridge] [4/15/2020, 16:52:14] Loaded plugin: "homebridge-securitysystem"
[Homebridge] [4/15/2020, 16:52:14] Registering accessory "homebridge-securitysystem.security-system"
[Homebridge] [4/15/2020, 16:52:15] Loaded plugin: "homebridge-tplink-smarthome"
[Homebridge] [4/15/2020, 16:52:15] Registering platform "homebridge-tplink-smarthome.TplinkSmarthome"
[Homebridge] [4/15/2020, 16:52:15] Loaded plugin: "homebridge-weather"
[Homebridge] [4/15/2020, 16:52:15] Registering accessory "homebridge-weather.Weather"
[Homebridge] [4/15/2020, 16:52:15] Loading 3 platforms...
[Homebridge] [4/15/2020, 16:52:15] [Camera ffmpeg] Initializing Camera-ffmpeg platform...
[Homebridge] [4/15/2020, 16:52:15] [SamsungTizen] Initializing SamsungTizen platform...
[Homebridge] [4/15/2020, 16:52:15] [TplinkSmarthome] Initializing TplinkSmarthome platform...
[Homebridge] [4/15/2020, 16:52:15] [TplinkSmarthome] homebridge-tplink-smarthome v4.0.1, node v12.13.1, homebridge v0.4.50
[Homebridge] [4/15/2020, 16:52:15] Loading 7 accessories...
[Homebridge] [4/15/2020, 16:52:15] [Roborock Vacuum] Initializing RoborockVacuum accessory...
[Homebridge] [4/15/2020, 16:52:15] [OpenWeatherMap Temperature] Initializing Weather accessory...
[Homebridge] [4/15/2020, 16:52:15] [Cloudiness] Initializing Weather accessory...
[Homebridge] [4/15/2020, 16:52:15] [Sun] Initializing Weather accessory...
[Homebridge] [4/15/2020, 16:52:15] [Sun] Starting Polling background service for Sun
[Homebridge] [4/15/2020, 16:52:15] [Today Min] Initializing Weather accessory...
[Homebridge] [4/15/2020, 16:52:15] [Today Max] Initializing Weather accessory...
[Homebridge] [4/15/2020, 16:52:15] Error: The requested accessory 'Security system' was not registered by any plugin.
    at API.accessory (/home/hoobs/.hoobs/node_modules/@hoobs/homebridge/lib/api.js:45:19)
    at Server._loadAccessories (/home/hoobs/.hoobs/node_modules/@hoobs/homebridge/lib/server.js:258:48)
    at Server.run (/home/hoobs/.hoobs/node_modules/@hoobs/homebridge/lib/server.js:78:14)
    at module.exports (/home/hoobs/.hoobs/node_modules/@hoobs/homebridge/lib/cli.js:90:12)
    at Object.<anonymous> (/home/hoobs/.hoobs/node_modules/@hoobs/homebridge/bin/homebridge:10:63)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
[Homebridge] [4/15/2020, 16:52:15] Got SIGTERM, shutting down service...
```

**Additional context**

**Comment by mkellsy** [comment]
We are testing a new release that fixes this

**Comment by mkellsy** [comment]
I just released 3.1.27
This issue has been fixed.
I recommend updating.
